### PR TITLE
Address Feedback

### PR DESCRIPTION
**Pull request by linuxwolf**
Addresses most of the feedback received, excluding any directory/file renames.

Notes:

* It was suggested to change instances of "I" to "We" in the templates, however that requires much more substantive changes to keep the messages understandable.
* It was suggested to rename the templates based on the escalation level they apply to.  That was not done in this PR in order to keep the content changes reviewable; a follow-up can be made to address that.

### CODE REVIEW

**Comment by dhruvdhody** [comment]
For Suresh's  comment on Level 2. I was wondering if it is worth adding 'pattern of abuse' to differentiate it with Level 1 clearly.  Something like - "An SAA team member sends an off-list message to the individual on behalf of the SAA team, with saa@ietf.org on cc. This message clearly identifies the concern and highlights the pattern of abuse. An offer to assist with re-framing language is also included. "

**Comment by dhruvdhody** [comment]
For the new README.md in email-templates - 

- The first message to the list is sent in Level 1, currently it is included in Level 0 for 'Unprofessional Commentary'. 

Also if we have this file, maybe no need to change filenames..

**Comment by dhruvdhody** [comment]
From first-message-to-list.txt, we can change [his|her] to their as well.

**Comment by dhruvdhody** [comment]
Thanks Mathew! Everything else looks good!

**Comment by linuxwolf** [comment]
Thanks @dhruvdhody!

File: sop.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ The sergeant-at-arms (SAA) team consists of the sergeants-at-arms and the IETF C
 
 The sergeants-at-arms strive to monitor all postings on ietf@ietf.org on a daily basis. The IETF Chair also monitors the list but may not be in a position to read each message or monitor every day.
 
-When a member of the SAA team sees a list posting that may be considered inappropriate according to [RFC 3005] and the [Unprofessional Commentary Description], the team member reports it to saa@ietf.org. Subscribers to ietf@ietf.org are also encouraged to report postings they believe to be inappropriate under those guidelines to saa@ietf.org. 
+When IETF discussion list subscribers believe that a posting that may be considered inappropriate according to [RFC 3005] or the [Unprofessional Commentary Description], they are encouraged to report it to saa@ietf.org. Please note that SAA team members are also subscribers to the IETF discussion list and they follow the same process to report posts that they consider inappropriate.
```
Comment (coopdanger) [inline sop.md]: I think we need to say RFC 3005 and the Unprofessional Commentary Description (not "or"). The two documents work together.

File: sop.md [hunk 2]
```diff
@@ -28,17 +28,17 @@ The SAA team member also requests that the individual take a voluntary 5-day "co
 
 ### Level 2: Temporary posting rights restriction
 
-An SAA team member sends an off-list message to the individual on behalf of the SAA team, with saa@ietf.org on cc. This message clearly identifies the concern and offers assistance with re-framing language.
+An SAA team member sends an off-list message to the individual on behalf of the SAA team, with saa@ietf.org on cc. This message clearly identifies the concern and offers assistance with re-framing language and highlights the pattern of abuse. An offer to assist with re-framing language is also included.
```
Comment (coopdanger) [inline sop.md]: This now has some grammar/repetition problem that should be fixed.

**Comment by coopdanger** [comment]
I think sticking with "I" in the templates is better, since we are signing "on behalf of."

I put my other comments in. Overall looks good. Thanks!

File: sop.md (hunk 1 above)
Comment (linuxwolf) [reply sop.md]: I think that commonly "and" means "all", where commonly "or" means "any".  As worded right now, to me I can see an interpretation of "the posting as to be inappropriate according to both/all sources, or it's considered appropriate", which is not the desired outcome.

If not "or", then how about "and possibly"?

Comment (coopdanger) [reply sop.md]: I think "or" is better than "and possibly." I can live with "or."

**Pull request merged**